pyqtgraph/pgcollections.py

```diff
@@ -10,15 +10,21 @@
   - ThreadsafeDict, ThreadsafeList - Self-mutexed data structures
 """
 
-import threading, sys, copy, collections
-#from debug import *
+import threading
+import sys
+import copy
 
 try:
     from collections import OrderedDict
 except ImportError:
     # fallback: try to use the ordereddict backport when using python 2.6
     from ordereddict import OrderedDict
-        
+
+try:
+    from collections.abc import Sequence
+except ImportError:
+    from collections import Sequence
+
```

Comment:
AFAIK pyqtgraph no longer supports py26, should we combine the Sequence and OrderedDict imports, also OrderedDict import is not caught

Reply:
Nevermind, OrderedDict is a concrete class not an ABC